### PR TITLE
Translations - Igbo

### PR DESCRIPTION
**Pull request by bruskiza**

### CODE REVIEW

**Comment by bruskiza** [comment]
@imsickofmaps @smn ready for review.

**Comment by imsickofmaps** [comment]
@bruskiza please also provide the json file by running https://www.npmjs.com/package/jspot against the new file too

**Comment by bruskiza** [comment]
@imsickofmaps like this?

File: config/go-ussd_public.ig.pot [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+"Project-Id-Version: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"PO-Revision-Date: 2016-04-25 07:59+0200\n"
+"X-Generator: Poedit 1.8.7\n"
+"Last-Translator: \n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: ig\n"
```
Comment (imsickofmaps) [inline config/go-ussd_public.ig.pot]: this should be `ibo_NG` and the json will need recreating as a result

**Comment by bruskiza** [comment]
@imsickofmaps and now?

**Comment by bruskiza** [comment]
@gsvr?

File: config/go-ussd_public.ibo_NG.pot [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+"Project-Id-Version: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"PO-Revision-Date: 2016-04-25 07:59+0200\n"
+"X-Generator: Poedit 1.8.7\n"
+"Last-Translator: \n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: ig\n"
```
Comment (imsickofmaps) [inline config/go-ussd_public.ibo_NG.pot]: There has been no change to this value in the latest commit

**Comment by imsickofmaps** [comment]
This file is full of non-ASCII characters:

``````
Proccessing go-ussd_public.ibo_NG.json
-------
Non-ascii translation found of <Please enter the number which is registered to receive messages. For example, 0803304899>: Biko tinye nomba ekwenti ohuru ị ga-acho iji na enweta ozi kwa izuka. Dịka ihe imatu 08033048990 
Non-ascii translation found of <We are sorry for your loss. You will no longer receive messages. Should you need support during this difficult time, please contact your local CHEW>: Odi anyi nwute n'ihe nfunari mere gi. ị ga-acho i nweta obere ozi ndi si na Hello Mama nwere ike inyere gị aka n'oge nsogbu a?
Non-ascii translation found of <Thank you. You language preference has been updated and you will start to receive messages in this language.>: Daalu. Asusu gị mmasị e emelitere na ị ga-amalite na-enweta ozi na asusu a.
Non-ascii translation found of <Sorry not a valid input. Welcome to Hello Mama. Please choose your language>: Ndo bughị a nti oso oso. Mmadu Bịa ka Hello Mama. Biko horo asusu gị
Non-ascii translation found of <Sorry not a valid input. Select:>: Ndo bughị a nti oso oso. Horo: 
Non-ascii translation found of <Thank you. You will now receive messages to support you during this difficult time.>: Daalu. I ga-now-enweta ozi na-akwado gị aka n'oge nsogbu a.
Non-ascii translation found of <Sorry not a valid input. Please tell us why you no longer want to receive messages so we can help you further.>: Ndo bughi a nti oso oso. Biko gwa anyi ihe mere i na-adikwaghi choro enweta ozi otú anyi nwere ike inyere gi n'ihu.
Non-ascii translation found of <Sorry, this number is already registered. They must opt-out before they can register again.>: Ndo, a onu ogugu a na-ama aha. Ha ga-apu na iso ha puru aha ozo. 
Non-ascii translation found of <Sorry not a valid input. You are already registered for baby messages.>: Ndo bughị a nti oso oso. Na ị na-Edebanyelarị n'ihi na nwa messages.
Non-ascii translation found of <What language would this person like to receive these messages in?>: Kedu asusu onye a choro iji na anata ozi ndị a?
Non-ascii translation found of <We will call twice a week. On what days would the person like to receive messages?>: Anyi ga-akpo ugboro abuo n’izu. Na ihe ubochi ga-onye-amasi na-enweta ozi?
Non-ascii translation found of <You are already registered for baby messages.>: Na ị na-Edebanyelarị n'ihi na nwa messages.
Non-ascii translation found of <Please tell us why you no longer want to receive messages so we can help you further.>: Biko gwa anyi ihe mere i na-adikwaghi choro enweta ozi otú anyi nwere ike inyere gi n'ihu.
Non-ascii translation found of <You have an incomplete registration. Would you like to continue with this registration?>: I nwere na-ezughị ezu ndebanye. I ga-acho ịnogide na a ndebanye?
Non-ascii translation found of <We're sorry, you do not have permission to update the preferences for this subscriber.>: Anyị nwute, ị na-adịghị ikike iji melite mmasị maka nke a utu magazin.
Non-ascii translation found of <Who will receive these messages?>: Onye ga-enweta ozi ndị a?
Non-ascii translation found of <We do not recognise this number. Please dial from the registered number or sign up with your local Community Health Extension worker.>: Anyị na-adịghị aghota a onu ogugu. Biko osịso si ughalaahia onu ogugu ma o bu banye na gị na mpaghara Community Health Extension oru.
Non-ascii translation found of <Who would you like to stop receiving messages?>: Onye ga-amasị gị ka ha kwusị anata ozi?
Non-ascii translation found of <Welcome to Hello Mama. Please choose your language>: Anyi na anabata gi na Hello Mama. Biko horo asusu gị
Non-ascii translation found of <Welcome to Hello Mama. Do you have permission to manage the number {{msisdn}}?>: Ndo bughị a nti oso oso. Mmadu Bịa ka Hello Mama. I nwere ikike jikwaa onu ogugu {{msisdn}}?
Non-ascii translation found of <Mobile number not registered.>: Mobile nomba adịghị aha
Non-ascii translation found of <Sorry not a valid input. Welcome to Hello Mama. Do you have permission to manage the number {{msisdn}}?>: Ndo bughị a nti oso oso. Mmadu Bịa ka Hello Mama. I nwere ikike jikwaa onu ogugu {{msisdn}}?
Non-ascii translation found of <Thank you. The number which receives messages has been updated.>: Imela. E melitela nomba nke na-anata ozi. 
Done Proccessing go-ussd_public.ibo_NG.json```
``````

**Comment by bruskiza** [comment]
@imsickofmaps: 

Ready for re-re-re-re--rihanna.

Nope. Review :D

File: config/go-ussd_public.ibo_NG.pot (hunk 2 above)
Comment (imsickofmaps) [inline config/go-ussd_public.ibo_NG.pot]: @bruskiza still not `ibo_NG` - fix and recompile JSON to inherit the fix.

**Comment by imsickofmaps** [comment]
@bruskiza 👍

**Pull request merged**